awx/main/models/schedules.py

```diff
@@ -97,40 +97,6 @@ class Meta:
         default={}
     )
 
-    # extra_data is actually a string with a JSON payload in it. This
-    # is technically OK because a string is a valid JSON. One day we will
-    # enforce non-string JSON.
-    def _clean_extra_data_system_jobs(self):
-        extra_data = self.extra_data
-        if not isinstance(extra_data, dict):
-            try:
-                extra_data = json.loads(self.extra_data)
-            except Exception:
-                raise ValidationError(_("Expected JSON"))
-
-        if extra_data and 'days' in extra_data:
-            try:
-                if type(extra_data['days']) is bool:
-                    raise ValueError
-                if float(extra_data['days']) != int(extra_data['days']):
-                    raise ValueError
-                days = int(extra_data['days'])
-                if days < 0:
-                    raise ValueError
-            except ValueError:
-                raise ValidationError(_("days must be a positive integer."))
-        return self.extra_data
-
-    def clean_extra_data(self):
-        if not self.unified_job_template:
-            return self.extra_data
-
-        # Compare class by string name because it's hard to import SystemJobTemplate
-        if type(self.unified_job_template).__name__ is not 'SystemJobTemplate':
-            return self.extra_data
-
-        return self._clean_extra_data_system_jobs()
-
```

Comment:
@chrismeyersfsu I wanted to ping you that I moved the logic that you recently added here to a method I introduce in this PR, is common among all UJT for prompts-related validation, `accept_or_ignore_variables` and `_ accept_or_ignore_variables`